### ISSUE TITLE
Some files & directories don't appear in Chocolat 2.3

### ISSUE THREAD

**Issue by cognitom** [fileability/chocolat-public#1619]
Such files and directories with specific names.

- `*.ttf` (file)
- `icon/` (directory)

![chocolat](https://cloud.githubusercontent.com/assets/16032/3806481/5498583e-1c4d-11e4-9553-a3911cc91350.png)

**Comment by tobiastom** [comment]
Still in Version 3.2.1.

**Comment by fileability** [comment]
Here is the full list so you can debate what should and should not be ignored:

```objc
static inline NSSet* CHBlacklistedExtensions(CHBlacklistClient cl) {
    NSMutableSet* s = [[NSMutableSet alloc] initWithObjects:@"o", @"so", @"dll", @"class", @"exe", @"pyc", @"elc", @"rbc",
                       @"7z", @"dmg", @"gz", @"iso", @"jar", @"rar", @"tar", @"zip",
                       @"tif", @"tiff", @"png", @"jpg", @"jpeg", @"gif", @"psd", @"ai", @"mov", @"aep", @"pdf", @"aif", @"aiff", @"ico", @"icns",
                       @"mp3", @"m4a", @"aif", @"aiff", @"ttf", @"bz2", @"hqx", 
                       @"lock", @"swp", @"out", @"tmp", 
                       @"mode1v3", @"pbxuser", @"classdescription",
                       nil];
    return s;
}

static inline NSSet* CHBlacklistedFilenames(CHBlacklistClient cl) {
    NSMutableSet* s = [[NSMutableSet alloc] initWithObjects:@".ds_store", @".ds_stor", @"ehthumbs.db", @"thumbs.db", @"ico", @"icon", nil];
    return s;
}

static inline NSSet* CHBlacklistedDirectoryExtensions(CHBlacklistClient cl) {
    NSMutableSet* s = [[NSMutableSet alloc] initWithObjects:@"xcodeproj", @"tmproj", @"esproj", @"chproj", @"nbproject", nil];
    return s;
}

static inline NSSet* CHBlacklistedExactDirectories(CHBlacklistClient cl) {
    NSMutableSet* s = [[NSMutableSet alloc] initWithObjects:@".fseventsd", @".spotlight-v100", @".temporaryitems", @".trashes", @".bzr", @".cdv", @".git", @".hg", @".pc", @".svn", @"cvs", @"rcs", @"sccs", @"_darcs", @"_sgbak", @"blib", @"_build", @".sass-cache", @"autom4te.cache", @"cover_db", @".metadata", nil];
    if (cl == CHBlacklistClient_QuickOpen) {
        [s addObject:@"build"];
        [s addObject:@"log"];
        [s addObject:@"tmp"];
        [s addObject:@"node_modules"];
    }
    return s;
}

static inline NSArray* CHBlacklistedSubstringDirectories(CHBlacklistClient cl) {
    NSMutableArray* s = [[NSMutableArray alloc] initWithObjects:@"~.dep", @"~.dot", @"~.nib", @"~.plst", nil];
    return s;
}
```

**Comment by tobiastom** [comment]
@fileability for me it would be enough if `CHBlacklistedFilenames ` does not match on directories. :)

**Comment by fileability** [comment]
@tobiastom I'll just take "icon", etc out. If you have .DS_Store as a directory you deserve what you get :stuck_out_tongue:

**Comment by tobiastom** [comment]
Agreed! :)

**Comment by fileability** [comment]
I will also remove "tmp", "_build", "cvs", "rcs", "sccs"

**Comment by tobiastom** [comment]
I can confirm it's fixed in 3.2.2. Thank you.
